python/private/pypi/pep508_env.bzl

```diff
@@ -30,7 +30,7 @@ _platform_system_values = {
     "windows": "Windows",
 }
 _sys_platform_values = {
-    "linux": "posix",
+    "linux": "linux",
```

Comment:
This change was required for `sys_platfomr==linux` otherwise all the different platforms are added to the list

Reply:
Yeah, this change is correct, thanks for fixing this. I noticed that too in one of my PRs.

Reply:
#2767 fixed this, could you please rebase?